### PR TITLE
Avoid fail when file does not exist

### PR DESCRIPTION
**Pull request by baturin**
Let's start sorting out issues from https://github.com/hatnote/montage/issues/141. That simple PR avoids montage from failure when importing renamed/removed/incorrect image files.

### CODE REVIEW

File: montage/loaders.py [hunk 1]
```diff
@@ -182,18 +184,25 @@ def get_entries_from_gsheet(raw_url, source='local'):
...
         files = []
         for filename in filenames:
             file_info = get_file_info(filename)
-            files.append(file_info)
+            if file_info is not None:
+                files.append(file_info)
+            else:
+                warnings.append(
+                    'file "%s" does not exist, please check that its name is spelled correctly, '
+                    'that it has not been renamed or removed' % (filename,)
```
Comment (mahmoud) [inline montage/loaders.py:200]: This is a good, actionable error message. However, we should keep an eye out for if there are many, many missing files. Assuming there are 0-10, I think the error display method defined in the JS above should be fine.

**Comment by baturin** [comment]
@mahmoud Yes, I think we should add some tests for situations like that (there are also several situations around that should be handled in a similar way). BTW, have you checked my previous test improvements https://github.com/hatnote/montage/pull/159?
As for displaying error message, that's also pretty good question, and I think yes, we should rework error display method to make it more structured and handle many errors (as there could be many files with issues like that). I'm going to think of a way once we handle all issues on backend.

**Comment by mahmoud** [comment]
Cool. So, what I'm understanding is that we can roll with this for now and will improve error display in a future PR?

As for #159, somehow I missed that! Left a comment. :)

**Comment by mahmoud** [comment]
@baturin just checking in on this, are we OK to merge + address errors in the next?

**Comment by baturin** [comment]
@mahmoud Yeah, let's continue improving error reporting in future PRs, and merge that one.

**Pull request merged**